src/Microsoft.Extensions.Testing.Abstractions/Messages/ProtocolVersionMessage.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Tools.Test
+{
+    public class ProtocolVersionMessage
```

Comment:
This has message in the name but has no relation to what seems like a message base class above. Seems confusing.